documentation/typescript.md

```diff
@@ -38,6 +38,7 @@ Add this comment to the top of your `dnsconfig.js` file:
 
 {% code title="dnsconfig.js" %}
 ```javascript
+/// <reference path="types-dnscontrol.d.ts" />
 // @ts-check
```

Comment:
The line above should probably say “add these comments” since there are two now. Also, I think `@ts-check` is supposed to be first.
```suggestion
// @ts-check
/// <reference path="types-dnscontrol.d.ts" />
```